packages/react-instantsearch-hooks/package.json

```diff
@@ -47,8 +47,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "algoliasearch-helper": "^3.7.4",
-    "instantsearch.js": "^4.40.1",
+    "algoliasearch-helper": "^3.8.0",
```

Comment:
.2 please, .0 is broken/deprecated

Reply:
I updated the peer deps for all packages in f6031759c30f7aa0cbf363b1bf305b44663c3511 then.